src/gf-platformid

```diff
@@ -41,8 +41,8 @@ coreos_gf_run_mount "${tmp_dest}"
 # * grub config
 # * BLS config (for subsequent config regeneration)
 # First, the grub config.
-# Set grub.cfg in case the arch uses anaconda
-if [ "$arch" == "ppc64le" ]; then
+# No grub on s390x
+if [ "$arch" != "s390x" ]; then
```

Comment:
Do we need this change? Didn't we already skip this on s390x?

Reply:
So, the master branch supports anaconda less install, but this is the 4.2 branch and anaconda is the only mode, so the all arches would use anaconda install and since s390x doesn't have grub, this change is to exclude it only for s390x.

Reply:
Ahh gotcha! I was under the impression the `rhcos-4.2` branch did support Anaconda-less installs for x86_64 at least. But yes, you're right.